staging/src/k8s.io/apimachinery/pkg/runtime/serializer/protobuf/protobuf.go

```diff
@@ -457,8 +459,10 @@ func (s *RawSerializer) Identifier() runtime.Identifier {
 	return rawSerializerIdentifier
 }
 
+// LengthDelimitedFramer is exported variable of type lengthDelimitedFramer
 var LengthDelimitedFramer = lengthDelimitedFramer{}
 
+// Provides length delimited frame reader and writer methods
```

Comment:
nit: why do we need to add comment for private variable here?
If adding, it would be better to write variable name like
```
lengthDelimitedFramer provides length delimited frame reader and writer methods
```